versions.tf

```diff
@@ -1,22 +1,18 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 0.14.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
       version = ">= 2.0"
     }
-    template = {
-      source  = "hashicorp/template"
-      version = ">= 2.0"
-    }
     null = {
-      source  = "hashicorp/null"
-      version = ">= 2.0"
+      source  = "hashicorp/random"
```

Comment:
looks like this provider is set twice, once as null and once as random

Reply:
corrected. it should be "hashicorp/null" for the null provider